src/nvim/version.c

```diff
@@ -972,6 +972,18 @@ void intro_message(int colon)
   int sponsor;
   char *p;
   static char *(lines[]) = {
+    N_("Welcome to Neovim!"),
+    "",
+    N_("First time using a vi-like editor?"),
+    N_("Type :Vimtutor<Enter> to get started!"),
+    "",
+    N_("Already know your way around Vim?"),
+    N_("See :help nvim for an introduction to Neovim."),
+    "",
+    N_("Can't find what you are looking for?"),
```

Comment:
I'm not satisfied with this one.. What about `Stuck and need help?` or `Still have questions?` or `Want to know more?`.